index.html

```diff
@@ -80,6 +80,33 @@
   <section id="sotd">
     <p>The Web Performance Working Group maintains a <a href="https://github.com/w3c/web-platform-tests/tree/master/html/webappapis/idle-callbacks">test suite</a> and an <a href="https://w3c.github.io/test-results/requestidlecallback/">implementation report</a> for the specification. Vendors interested in implementing this document SHOULD join the mailing lists below and take part in the discussions.</p>
   </section>
+  <section>
+    <h2>Dependencies</h2>
+    <p>The terms <dfn data-cite="html51#sec-browsing-contexts">browsing context
+    </dfn>, <dfn data-cite="html51#event-loop">event loop</dfn>,
+    <dfn data-cite="html51#event-loops-processing-model">event loop processing
+    model</dfn>, <dfn data-cite="html51#spinning-the-event-loop">spin the event
+    loop</dfn>, <dfn data-cite="html51#fully-active">fully active</dfn>,
+    <dfn data-cite="html51#tasks">tasks</dfn>,
+    <dfn data-cite="html51#task-source">task source</dfn>,
+    <dfn data-cite="html51#task-queues">task queues</dfn>,
+    <dfn data-cite="html51#queuing">queue a task</dfn>,
+    <dfn data-cite="html51#microtask-queue">microtask queue</dfn>,
+    <dfn data-cite="html51#list-of-active-timers">list of active timers</dfn>,
+    <dfn data-cite="html51#dom-windowtimers-settimeout">setTimeout</dfn>,
```

Comment:
wrap these ones in `<code>` and see below... then you don't need `<code>` anywhere else and you the spec will be auto-linked correctly.